lib/scout_apm/instruments/net_http.rb

```diff
@@ -22,12 +22,17 @@ def install
             include ScoutApm::Tracer
 
             def request_with_scout_instruments(*args,&block)
-              url = (@address + args.first.path.split('?').first)[0..99]
-              self.class.instrument("HTTP", "request", :desc => url) do
+              self.class.instrument("HTTP", "request", :desc => request_scout_description(args.first)) do
                 request_without_scout_instruments(*args, &block)
               end
             end
 
+            def request_scout_description(req)
+              path = req.path
+              path = path.path if path.respond_to?(:path)
```

Comment:
Yes, see https://github.com/ruby/ruby/blob/v2_1_9/lib/net/http/generic_request.rb#L23 the `Addressable::URI` is assigned to path. Calling `path` returns the `Addressable::URI` not the expected string.